.gitignore

```diff
@@ -29,7 +29,18 @@ pnpm-debug.log
 pnpm-debug.log.*
 yarn-error.log
 yarn-error.log.*
+.DS_Store
 .env
+.idea
+.npmrc
+.vscode/*
+!.vscode/extensions.json
+*.local
+*.njsproj
+*.ntvs*
+*.sln
+*.suo
+*.sw?
```

Comment:
I'm not fond of adding all this stuff, the ignore file can be infinite this way.

What do you think of adding only what's necessary?

Reply:
I think they can be removed and then added back on a case by case basis if necessary.